src/js/index.js

```diff
@@ -466,10 +466,19 @@ export const hpe = deepFreeze({
       },
       size: 'xsmall',
       color: 'text',
-      margin: {
-        start: 'none',
-      },
+      margin: 'none',
     },
+    extend: `
+      // This selector will work when label or help
+      // are passed strings. However, it will be too
+      // broad in situations where a node is passed.
+      // The proper solution should be supporting a
+      // formField.input.margin or formField.input in 
+      // the theme.
+      > div:first-of-type {
+        margin: 4px 0px;
```

Comment:
It seems like what we really want is to allow formField to have a margin-top defined and then help and label would both have no margin. That way, regardless of if just a label or label + help were provided, the correct spacing between text and formField would apply. This would be a grommet change though because I don't think that's currently supported.

Reply:
This does look like the kind of extend behavior that we'd like to not need.

Reply:
Correct and agree.

Was primarily using this as documentation of my investigation so that it could be referenced for Grommet enhancement.

Reply:
yep, well done 👍